.jenkins/cscs-perftests/batch.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash -l
+
+# Copyright (c) 2020 ETH Zurich
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+# Computes the status of the job and store the artifacts
+status_computation_and_artifacts_storage() {
+    ctest_exit_code=$?
+    ctest_status=$(( ctest_exit_code + configure_build_errors + test_errors + plot_errors ))
+
+    # Copy the testing directory for saving as an artifact
+    cp -r ${build_dir}/Testing ${orig_src_dir}/${configuration_name}-Testing
+    cp -r ${build_dir}/reports ${orig_src_dir}/${configuration_name}-reports
+
+    echo "${ctest_status}" > "jenkins-hpx-${configuration_name}-ctest-status.txt"
+    exit $ctest_status
+}
+
+# Args for the pyutils suite
+logfile=jenkins-hpx-${configuration_name}.out
```

Comment:
This looks like it's the same filename as the logfile created by slurm. Is that intentional?

Reply:
Thanks, was using it when I wasn't using sbatch and forgot to rename it!